fastlane/swift/SocketClient.swift

```diff
@@ -193,13 +215,15 @@ extension SocketClient: StreamDelegate {
             // Still getting response from server eventhough we are done.
             // No big deal, we're closing the streams anyway.
             // That being said, we need to balance out the dispatchGroups
+            print("Leave \(#function) \(#line)")
             self.dispatchGroup.leave()
             return
         }
         
         if aStream === self.inputStream {
             switch eventCode {
             case Stream.Event.openCompleted:
+                print("Leave \(#function) \(#line)")
```

Comment:
Either delete or 👇 ?

```suggestion
                verbose(message: "Leave \(#function) \(#line)")
```